bin/benchmark_elbo.jl

```diff
@@ -12,7 +12,7 @@ const dat_dir = joinpath(Pkg.dir("Celeste"), "dat")
 srand(1)
 println("Loading data.")
 
-S = 100
+S = 3
```

Comment:
Would you try adding `sortedby=:count` to the call to `Print.profile` ? Sometimes bottlenecks really "pop out" once when you view the lines sorted by run times. I'm glad to take a look too once you push the changes to `benchmark.jl`.